jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ZipkinSender.java

```diff
@@ -101,15 +101,15 @@ public int append(Span span) throws SenderException {
     byte[] next = encoder.encode(backFillHostOnAnnotations(span));
     int messageSizeOfNextSpan = delegate.messageSizeInBytes(Collections.singletonList(next));
     // don't enqueue something larger than we can drain
-    if (Integer.compare(messageSizeOfNextSpan, delegate.messageMaxBytes()) > 0) {
+    if (Integer.valueOf(messageSizeOfNextSpan).compareTo(delegate.messageMaxBytes()) > 0) {
```

Comment:
how is this different from `messageSizeOfNextSpan > delegate.messageMaxBytes()` ?